processor/processor.go

```diff
@@ -1490,8 +1492,7 @@ func (proc *Handle) processJobsForDest(partition string, subJobs subJob) *transf
 			},
 			)
 			if proc.config.eventSchemaV2Enabled && // schemas enabled
-				// source has schemas enabled or if we override schemas for all sources
-				(source.EventSchemasEnabled || proc.config.eventSchemaV2AllSources) &&
```

Comment:
Please delete `SourceT#EventSchemasEnabled` in `backendconfig/types.go` and adapt `processor_test.go`